dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype-mbeans.properties

```diff
@@ -48,6 +48,9 @@ plugin.http.endpoint.desc=Specify the identifier of the http endpoint to include
 plugin.connectTimeout.name=Connection timeout
 plugin.connectTimeout.desc=Identifies the maximum amount of time that the application server should maintain a connection with the web server.
 
+plugin.persistTimeOut_Reduction.name=Reduce PersistTimeOut for ConnectionTTL
+plugin.persistTimeOut_Reduction.desc=Identifies the amount of time to subtract from the PersistTimeOut value for ConnectionTTL.  
+
```

Comment:
`Identifies the amount of time to subtract from the PersistTimeOut value for ConnectionTTL.`
suggested edit:

`Specifies the amount of time to subtract from the value of the persistTimeOut parameter to calculate the value for the ConnectionTTL parameter.`